docs/admin/multiple-zones.md

```diff
@@ -73,6 +73,18 @@ available and can tolerate the loss of a zone, the control plane is
 located in a single zone.  Users that want a highly available control
 plane should follow the [high availability](/docs/admin/high-availability) instructions.
 
+* StatefulSet volume zone spreading when using dynamic provisioning is currently not compatible with
+pod affinity or anti-affinity policies.
+
+* StatefulSet volume zone spreading does not work if the StatefulSet name
```

Comment:
There was some work done to correct this. I would go with something like "If the name of the StatefulSet contains dashes ("-"), volume zone spreading may not provide a uniform distribution of storage across zones"